script.js

```diff
@@ -10,9 +10,31 @@ function renderCalendarCells() {
         }
     });
 }
+function openEventCreationModal() {
+    const modal = document.getElementById("eventModal");
+    modal.style.display = "block";
+}
+function closeEventCreationModal() {
+    const modal = document.getElementById("eventModal");
+    modal.style.display = "none";
+}
+function initEventListeners() {
+    const btn = document.querySelector(".add-event-button");
+    const span = document.getElementsByClassName("close")[0];
```

Comment:
```suggestion
    const closeModalButton = document.getElementById("closeModal");
```
I suggest selecting more specifically, ie. by id, since if you by any chance will add another close button with same class above this one, logic will break. Or you could use querySelector to select by class.